effekt/shared/src/main/scala/effekt/RecursiveDescent.scala

```diff
@@ -124,6 +137,7 @@ class RecursiveDescent(positions: Positions, tokens: Seq[Token], source: Source)
   def skip(): Unit =
     previous = tokens(position)
     position += 1;
+//    currentLabel = None // TODO(jiribenes, 2025-05-20): ???
```

Comment:
I thought that this should be set to `None` once we consume a token, but it doesn't work (I get "expected identifier got `}`" for the example problem above)